docs/how-to-release.md

```diff
@@ -0,0 +1,193 @@
+    - Write a description following the template:
+    
+      ```markdown
+      [All changes](https://github.com/opendatakit/briefcase/compare/v1...v2)
+      
+      **Highlights**
+      - Some highlighted feature
+      - Some highlighted technical improvement
+      - Some highlighted bugfix
+      - Some important notice about breaking changes
+      
+      **Added**
+      - #123 Title of issue 123
+        - Author of the solution of #123
+      
+      **Removed**
+      - #123 Title of issue 123
+        - Author of the solution of #123
+      
+      **Fixed**
+      - #123 Title of issue 123
+        - Author of the solution of #123
+      ```
+      
+      Notes:
+      - Replace the versions of the `All changes` link with the previous and current tags e.g. `v1.11.3...v.1.12.0`
+      - In the `Added`, `Removed`, and `Fixed` sections, always link in this order: Issue > PR > Commit
+      - The `Added` list would compile every new added feature
+      - The `Removed` list would compile every removed feature
+      - The `Fixed` list would compile bug fixes
+      - An easy way to get a comprehensive list of items for these lists consists on searching on the `All changes` diff the terms `Merged` and ` and `. The second term will reveal merges by more than one author which indicate a "squash merge".
+      - The list of items should be sorted in the same order as they've been closed
+    
+3. Once the release information has been completed, click on the `Publish release` button.
+
+    GitHub will create a tag from the selected branch
+  
+4. Go to your releases workspace and sync your local repo with `git pull`
+
+    You should see a message telling you about the new tag GitHub has created for your release
+
+5. Check out the tag with `git checkout vx.y.z`. (replace x.y.z with the actual numbers you've used in step 2)
+
+    Git should announce that you're on a `detached HEAD`. That's OK.
+    
+6. Verify that logging, error monitoring, and user tracking have production configurations
+
+   **Logging configuration**
+   
+   Copy the file at `res/logback.xml.example-release` to `res/logback.xml`
+   
+   **Error monitoring, and user tracking configuration**
+   
+   Create the file `gradle.properties` at the root of the project and add these lines:
+   
+   ```groovy
+   sentry.enabled=true
+   googleAnalytics.trackingId={TRACKING_ID}  
+
+   ```
+   
+   Use the Google Analytics tracking ID obtained in the [Google Analytics Dashboard](https://analytics.google.com):
+   
+   - Select the `ODK Briefcase` app
+   - Go to `Admin > (Property section) Property Settings`
+   
+   Tracking IDs follow the pattern `UA-12345678-1`
+     
+  
+7. Build the release JAR file with `./gradlew clean build -xtest`
+
+    The JAR file will be located at `build/libs` and the filename should be like: `ODK Briefcase vx.y.z.jar`. (replace x.y.z with the actual numbers you've used in step 2)
```

Comment:
Maybe our Gradle task should just do that and we can drop the text under 8.

Reply:
I went ahead and changed the build file to fix this